shell/assets/translations/en-us.yaml

```diff
@@ -237,6 +237,25 @@ about:
     title: Image Lists
   downloadCLI:
     title: CLI Downloads
+  diagnostic:
+    title: Diagnostic
```

Comment:
This is probably better as a plural - `Diagnostics` (`Diagnostic Page` would be correct, but we can rightly drop the page here)